index.js

```diff
@@ -4,7 +4,41 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let command = process.argv[2]
+    const numbers = []
+    let total = 0
+    let count = 0
 
+    // Edge cases that return errors when an invalid command is given
+    if (command !== 'plus' && command !== 'minus') {
+        if (command === undefined) {
+            return 'No operation provided...'
+        }
+        return `Invalid operation: ${command}`
+    }
+
+    if (process.argv.length < 4) {
+        return 'No numbers provided...'
+       }
+
+    // Adds to total if the operator is plus and subtracts from total
+    // if the operator is minus. The total must be intitialized as the first
+    // given argument to ensure it actually finds the difference of the numbers
+    for (i = 3; i < process.argv.length; i++) {
+        numbers.push(Number(process.argv[i]))
```

Comment:
creating a new array and pushing the numbers there to then add them is a little redundant. I think if you convert them to a number from `process.argv[i]` and then add them to `total`, you can achieve the same outcome in less steps.